config.md

```diff
@@ -25,6 +25,8 @@ For example, if an implementation is compliant with version 1.0.1 of the spec, i
 **`root`** (object, required) configures the container's root filesystem.
 
 * **`path`** (string, required) Specifies the path to the root filesystem for the container.
+  The path can be an absolute path (starting with /) or a relative path (not starting with /), which is relative to the bundle. e.g.
+  If the absolute path of root filesystem is /to/bundle/rootfs, we can set it or rootfs to`path.
```

Comment:
On 09/09/2016 12:39 AM, W. Trevor King wrote:
> How about: For example, with a bundle at `/to/bundle` and a root filesystem at `/to/bundle/rootfs`, the `path` value can be either `/to/bundle/rootfs` or `rootfs`.
It looks better than mine. Fixed.